test/buildah-bud/buildah-tests.diff

```diff
@@ -36,20 +36,7 @@ index 166316d5..775d7c9b 100644
  }
  
  function starthttpd() {
-@@ -84,6 +101,12 @@ function teardown(){
- function teardown_tests() {
-     stophttpd
- 
-+    if [[ -n "$PODMAN_SERVER_PID" ]]; then
-+        kill $PODMAN_SERVER_PID
-+        wait $PODMAN_SERVER_PID
-+        rm -f /run/podman/podman.sock
-+    fi
-+
```

Comment:
I'm pretty sure you want to keep this, too. This stops the `podman system service` process that is run for `podman-remote build` tests.